mantidproject/newstarter
# Conor Finn Exercises

**Description of Work**
Solution to exercise 01 of the new starter C++ exercises. 

**To test:**
1. Create binaries
2. Run from command line. 
3. When prompted, enter the path for the file to count.
4. When prompted, enter the path for an output file. 
5. Check the output file specified for accuracy. 

2019-07-16:
Completes #45
